packages/image_picker/image_picker_platform_interface/lib/src/types/multi_image_picker_options.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Specifies options for picking multiple images from the device's gallery.
+class MultiImagePickerOptions {
```

Comment:
Ideally we would share this with `ImagePickerOptions`, and extract a shared class that has the elements that are image-specific (i.e., everything here), and then an outer class with anything else. We don't need to do another change to the singe-image version now, but we should prep for it by structuring this that way: all of the fields here should move to a new `ImageOptions`, and then `MultiImagePickerOptions` should have one `ImageOptions` field. Eventually we can switch the single-image version to use `ImageOptions` instead of these four fields too.